src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -303,7 +303,8 @@ impl<'a, 'tcx> Qualifier<'a, 'tcx, 'tcx> {
                 TerminatorKind::Goto { target } |
                 TerminatorKind::Drop { target, .. } |
                 TerminatorKind::Assert { target, .. } |
-                TerminatorKind::Call { destination: Some((_, target)), .. } => {
+                TerminatorKind::Call { destination: Some((_, target)), .. } |
+                TerminatorKind::FalseEdges { real_target: target, .. } => {
```

Comment:
This should be handled like an `if false` aka a `SwitchInt`. I believe we'll get rid of this soon for MIRI, but try to be consistent until then.